packages/apps/composer-app/src/playwright/app-manager.ts

```diff
@@ -60,6 +60,17 @@ export class AppManager {
     return this.page.getByTestId(`${plugin}.createObject`).last().click();
   }
 
+  async createFolder() {
+    await this.page.getByTestId('spacePlugin.createObject').last().click();
+    return this.page.getByTestId('spacePlugin.createFolder').last().click();
+  }
+
+  async deleteObject(itemNumber: number) {
```

Comment:
maybe make this `deleteFolder` and use the `testId` of `spacePlugin.folder`?

make another one for `deleteObject`?